githooks/post-commit

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+REPO_DIR=$(git rev-parse --show-toplevel)
+SRC_PATH="$REPO_DIR/exasol_sagemaker_extension"
+
+if [ -a .commit ]
+  then
+  rm .commit
+  git add "$SRC_PATH/resources/"
```

Comment:
you should be able to do this also in the precommit hook. The point where the precommit sometimes makes a bit trouble is a rebase, but not sure if this can be solved via a post commit hook

Reply:
I could not add the change to the same commit with the pre-commit. but post-commit enable me  adding the generated file to the current commit.

Reply:
hm, strange, it usually worked for us. Did you have a git add in your precommit hook